### DISCUSSION TITLE
Name or Label Item?

### DISCUSSION THREAD

**Discussion by Tony4CEOs** [hluk/CopyQ#1571] [Q&A]
Right now, each copied item shows in the list as its contents. 
1. Is there a setting to display the items as just their first line, so more items show in a window of a given size?
2. Is it possible to give items meaningful names? For example, I have items containing individual telephone numbers. It would be useful if I could label each with the name of the person whose number it is without changing what gets pasted when that number is selected.

**Comment by hluk** [comment]
>     1. Is there a setting to display the items as just their first line, so more items show in a window of a given size?

See "Show simple items" option in History config tab
 
>     2. Is it possible to give items meaningful names? For example, I have items containing individual telephone numbers. It would be useful if I could label each with the name of the person whose number it is without changing what gets pasted when that number is selected.

Shift+F2 to add a note to selected item.

**Comment by Tony4CEOs** [comment]
Dear Lukas,

Thank you for this information. I’m pleased and surprised that the support on this free tool is better than what I get for some software I've paid a great deal to use. I also apologize for asking questions that are easily answered using the documentation and exploring the menus.
Best wishes,
Tony Mayo
Executive Coach
202.905.2588
[cid:image001.png@01D6FBD0.AB8B70E0]<https://tonymayo.com/keynote-speeches-tony-mayo/>
